Bixal/move.mil
# Feature/156547892/theme back to top button

Theming for "Return to Top" button, including a template change moving it from the footer to the main section. Executes https://www.pivotaltracker.com/story/show/156547892

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Changes the page.html.twig template to move the 'Return to Top' link from the footer to the main section, just outside the grid to allow it to float all the way right. Also changed link wrapper classes to match live site code.
- Copy/pasted live site's sass code (`app/assets/stylesheets/components/_layout.scss` lines 71-100) into new partial
- Added new logic to support consistent placement for logged-in users 

## Testing

To verify the changes proposed in this pull request…

1. Pull code
1. Run `npm run build` from theme folder to compile new sass
1. Go to any site page with enough content to require scrolling. The link should appear similarly to the live site and stay just below the top of the window when scrolled down the page.
1. If desired, test the link behavior both logged in and logged out and at different screen sizes

## Screenshots

Live site - top of page:
<img width="998" alt="screen shot 2018-04-09 at 3 12 43 pm" src="https://user-images.githubusercontent.com/29130580/38517606-85c0a202-3c08-11e8-942f-9ca5d351acb1.png">

Live site - scrolled down page:
<img width="1002" alt="screen shot 2018-04-09 at 3 13 30 pm" src="https://user-images.githubusercontent.com/29130580/38517640-9c918028-3c08-11e8-933b-1ca9d22fe805.png">

Local with change - top of page:
<img width="873" alt="screen shot 2018-04-09 at 3 14 14 pm" src="https://user-images.githubusercontent.com/29130580/38517679-b33463fe-3c08-11e8-836a-ffba411d5df5.png">

Local with change - scrolled down:
<img width="867" alt="screen shot 2018-04-09 at 3 14 56 pm" src="https://user-images.githubusercontent.com/29130580/38517713-cd696076-3c08-11e8-93ba-c7884a44900b.png">
